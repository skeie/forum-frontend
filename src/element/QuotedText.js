//@flow
import React from 'react';
import { View } from 'react-native';
import { Text, theme } from '../common';

function QuotedText({ text }: { text: string }) {
    console.log(text, 'sapdap text');
    return (
        <View
            style={{
                backgroundColor: theme.primary,
                padding: 10,
                borderRadius: 10,
                marginTop: 10,
            }}>
            <Text style={{ color: 'white', fontSize: 14 }} type="date">
                {text}
            </Text>
        </View>
    );
}

export default QuotedText;
