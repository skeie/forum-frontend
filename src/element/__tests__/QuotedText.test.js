import React from 'react';
import QuotedText from '../QuotedText';
import renderer from 'react-test-renderer';

it('Username renders correctly', () => {
    const tree = renderer
        .create(<QuotedText text="www.jegerbest.com" />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
