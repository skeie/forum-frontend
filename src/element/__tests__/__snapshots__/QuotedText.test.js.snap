// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Username renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#FD8C80",
      "borderRadius": 10,
      "marginVertical": 10,
      "padding": 10,
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    disabled={false}
    ellipsizeMode="tail"
    style={
      Array [
        Object {
          "color": "white",
          "fontFamily": "poppins",
        },
        Object {
          "color": "#DCDDE3",
          "fontFamily": "italic",
        },
        Object {
          "color": "white",
          "fontSize": 14,
        },
      ]
    }
    type="date"
  >
    www.jegerbest.com
  </Text>
</View>
`;
